pkg/controller/bundle/bundle_unpacker_test.go

```diff
@@ -77,7 +77,7 @@ func TestConfigMapUnpacker(t *testing.T) {
 		args        args
 		expected    expected
 	}{
-		{
+		/*{
```

Comment:
is this test no longer needed? maybe we should either remove, or if there's something we need to track, create an issue and tag it here?